tests/simplex/step-03.cc

```diff
@@ -0,0 +1,297 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 1999 - 2020 by the deal.II authors
+ *
+ * This file is part of the deal.II library.
+ *
+ * The deal.II library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE.md at
+ * the top level directory of deal.II.
+ *
+ * ---------------------------------------------------------------------
+
+ *
+ * Authors: Wolfgang Bangerth, 1999,
+ *          Guido Kanschat, 2011
+ */
```

Comment:
Could you have a look at the header of https://github.com/dealii/dealii/blob/622107e2928bbe96110393da1c5bd986ba9b707b/tests/simplex/step-02.cc#L1-L22. In particular, could you make some notes if you had some issues while porting the tutorial.